README.md

```diff
@@ -30,6 +30,19 @@ $ rake db:migrate
 $ rake server
 ```
 
+### Loading sample data
+Clear your solr index with:
+```bash
+$ rake lae:drop_index
+```
+
+Then index sample data with:
+```bash
+$ rake lae:reindex
+```
+Note: This will take about an hour to finish indexing all data, but data will show up in your index after
```

Comment:
it will take an hour to index the sample data?

Reply:
It’s pulling all 12k records from production right now, so a smaller local dataset will fix this.

Reply:
Before this change it was pulling from fixtures I believe and it was pretty quick. Is this no longer acceptable? Could we keep both options?

Reply:
I've added 50 sample JSONLD files and added a Rake task to index them.  I've left the other Rake task for indexing from production.

Reply:
awesome, thank you!!